packages/components/data-table-manager/src/data-table-manager.tsx

```diff
@@ -10,7 +16,7 @@ import type {
 import { useDataTableManagerContext } from '@commercetools-uikit/data-table-manager/data-table-manager-provider';
 
 const DataTableManager = <Row extends TRow = TRow>(
-  props: TDataTableManagerProps<Row>
+  props: PropsWithChildren<TDataTableManagerProps<Row>>
```

Comment:
Could you maybe explain the need for using the `PropsWithChildren` type?
I'm asking because the `TDataTableManagerProps` type already defines the `children` property.

Reply:
fixed in [79e91d7](https://github.com/commercetools/ui-kit/pull/3034/commits/79e91d72e7be9ce8dc00a911b81e8a67f566ba0b)